Added 09/25 Upcoming Releases Update

russjones opened the pull request.

russjones committed:
Added 09/25 Upcoming Releases Update

r0mant committed:
Update upcoming-releases.mdx

r0mant reviewed (approved):
lgtm once remaining todo items are filled in

zmb3 committed:
Add new feature details

Added descriptions for the Web UI Workload ID and Relay Service features in the upcoming releases documentation.
